tsstats/templates/index.jinja2

```diff
@@ -49,5 +59,23 @@
   {% endfor %}
   <small>Generated by <a href="https://github.com/Thor77/TeamspeakStats" rel="noopener">TeamspeakStats</a> at {{ creation_time|frmttime }}</small>
 </div>
+
+<script type="text/javascript">
+  document.addEventListener('click', function(event) {
+    var toggler = event.target;
+    if (toggler.dataset.toggle !== 'collapse') {
+      return;
+    }
+    var toggleTarget = document.getElementById(toggler.dataset.target);
+
+    if (toggleTarget.classList.contains('collapse')) {
+      toggleTarget.classList.remove('collapse');
```

Comment:
Huh, please don't mess with the `collapse`-class here. You can check the current collapse-state via the `aria-expanded`-attribute, can't you?